gamemode/modules/hunger/sh_hunger.lua

```diff
@@ -0,0 +1,32 @@
+local PlayerMeta = FindMetaTable( "Player" )
+
+function PlayerMeta:GetHunger()
+    return self:GetNWInt("rvr_hunger")
+end
+
+function PlayerMeta:SetHunger(n)
+    self:SetNWInt("rvr_hunger", n)
+end
+
+function PlayerMeta:AddHunger(n)
+    local hunger = self:GetNWInt("rvr_hunger")
+    hunger = math.max( hunger+n, 0 )
+    self:SetNWInt("rvr_hunger", hunger)
+end
+
+
+function PlayerMeta:GetThirst()
+    return self:GetNWInt("rvr_thirst")
+end
+
+function PlayerMeta:SetThirst()
+    self:SetNWInt("rvr_thirst")
+end
+
+function PlayerMeta:AddThirst(n)
```

Comment:
I dont like calling this thirst and hunger. this value isnt their Thirst/Hunger its their Water/Food. Calling it "Food" and "Water" seems weird though, any ideas

Reply:
I think `AddFood` and `AddWater` is fine?